float64 sigma_l
float64 sigma_a
float64 sigma_b
---
float64 l
float64 a
float64 b
